ipaplatform/fedora/paths.py

```diff
@@ -36,6 +36,8 @@ class FedoraPathNamespace(RedHatPathNamespace):
     NAMED_CRYPTO_POLICY_FILE = "/etc/crypto-policies/back-ends/bind.config"
     if HAS_NFS_CONF:
         SYSCONFIG_NFS = '/etc/nfs.conf'
+    DNSSEC_OPENSSL_CONF = "/etc/ipa/dnssec/openssl.cnf"
+    DNSSEC_KEYFROMLABEL = "/usr/sbin/dnssec-keyfromlabel"
```

Comment:
shouldn't these be in base/paths.py?

Reply:
They are already in
https://github.com/freeipa/freeipa/blob/b65fed7c9de754ac653543c61ab725bad3a0e1e1/ipaplatform/base/paths.py#L235
https://github.com/freeipa/freeipa/blob/b65fed7c9de754ac653543c61ab725bad3a0e1e1/ipaplatform/base/paths.py#L70

Reply:
If you mean defaults, then I think the defaults should be RHEL based.
Maybe I'm wrong.

Reply:
yeah, you're right, this is fine